Week2/part1.js

```diff
@@ -0,0 +1,81 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world'
+});
+
+connection.connect();
+//What is the capital of country X ? (Accept X from user)
+function capitalOfCountry(country) {
```

Comment:
These functions should be async. Please take a look at the examples I posted in slack. ;)